### DISCUSSION TITLE
Accessing resolvedUrl from getServerSideProps context parameter

### DISCUSSION THREAD

**Discussion by Sosodope** [vercel/next.js#28830] [Help] [answered]
I saw in the docs that you can get the request url without _next/data prefix but I haven't had any luck trying to access the resolvedUrl. Anyone have any luck with that.

**Comment by thepuzzlemaster** [comment, marked as answer]
I had a similar problem, and the discussion in https://github.com/vercel/next.js/discussions/15787, led me to find that `context.resolvedUrl` will give you this info. I should be clear that in my case, I needed the resolvedUrl in `getServerSideProps`. Not sure where you're trying to get it from.

**Reply by Sosodope** [reply]
I also need it in the getServerSideProps. Thanks for this, will check out the other discussion

**Reply by Sosodope** [reply]
My context.resolvedUrl in the getServerSideProps is undefined. So I just wrote a function to build the url taking the req.host and joining all the entries in the query object. That seems to do what I need for now. Anyways, appreciate the link to the post. The answer there was helpful

**Reply by thepuzzlemaster** [reply]
Yeah, I had considered that as well, but was really hoping to avoid having to do that. I wonder if I'm using a newer version of nextjs perhaps? 

I added a console.log(context) to my `getServerSideProps` function to eventually find it.

**Reply by Sosodope** [reply]
oh yeah, you were right. I upgraded to the latest next version and it seems to give me the resolvedUrl 😅

**Comment by thepuzzlemaster** [comment]
Yay! Everyone loves a happy ending :)
